services/QuillLMS/app/queries/scorebook/query.rb

```diff
@@ -118,7 +118,7 @@ def self.date_conditional_string(begin_date, end_date, offset)
   end
 
   def self.to_offset_datetime(date, offset)
-    (Date.parse(date).midnight - offset.seconds).to_s(:db)
+    (Date.parse(date).midnight - offset.seconds).to_fs(:db)
```

Comment:
I believe that we also use `to_s(:db)` in `app/queries/snapshots/period_query.rb`.  Not sure if that's the same issue as here, but figured I'd flag it.

Reply:
Good call.  I was trying to figure out why there were no deprecation warnings after this fix, then I remembered that I was not looking at the correct [configuration](https://app.circleci.com/pipelines/github/empirical-org/Empirical-Core/23417/workflows/4c498ea0-d9c9-438e-99b2-f21ad04e67ac/jobs/280247?invite=true#step-108-1681)
(over 1000 warnings!)